courses/machine_learning/deepdive2/structured/solutions/5a_train_keras_ai_platform_babyweight.ipynb

```diff
@@ -0,0 +1,1593 @@
+{
```

Comment:
<p>you need to include </p><p>os.environ["PROJECT"] = PROJECT</p><p>here or else later cells won't recognize the environment variable.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/674/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Same as before, export takes care of project.</p>